common/scripts/.githooks/make_lint-all.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Launches fmt and lint checks
+make lint-all 
```

Comment:
@itdove can you help kill the red icon at the end? 
![image](https://user-images.githubusercontent.com/4461983/68351515-ac632400-013e-11ea-93e5-286c9ddfd303.png)